.travis.yml

```diff
@@ -12,8 +12,6 @@ env:
   - TOXENV=django110
 
 matrix:
-  allow_failures:
```

Comment:
I assume that this means that Tox will now run tests againsst Python 3 too. I don't know enough about the combination of Travis and Tox to be sure. Is there a way to confirm this from the Travis build reports?